src/include/libpmemfile-core.h

```diff
@@ -60,12 +60,19 @@ void pmemfile_pool_close(PMEMfilepool *pfp);
 
 PMEMfile *pmemfile_open(PMEMfilepool *pfp, const char *pathname, int flags,
 		...);
+PMEMfile *pmemfile_openat(PMEMfilepool *pfp, PMEMfile *file,
```

Comment:
Because pmemfile-core does not deal with file descriptors. It's a job for the system integration layer. If pmemfile-core would use file descriptors it would make a illusion of them being usable for syscalls.